06-data-formats-caveats.md

```diff
@@ -6,6 +6,11 @@ root: .
 
 Materials by: **Jeffrey Hollister**, **Alexander Duryee**, **Jennifer Bryan**, **Daisie Huang**, **Ben Marwick**, **Christie Bahlai**, **Owen Jones**, **Aleksandra Pawlik**
 
+## Learning Objectives
+* Identify problems with using the .csv file format.
+* Apply best practices for data cleaning to avoid problems with the .csv file format.
+
```

Comment:
That section isn't filled out yet, but I would also add something like

Determine if an alternate file format, such as tsv, would be more appropriate for some data